code/modules/atmospherics/machinery/atmospherics.dm

```diff
@@ -298,8 +298,10 @@ Pipelines + Other Objects -> Pipe network
 		else if(target_move.can_crawl_through())
 			if(returnPipenet() != target_move.returnPipenet())
 				user.update_pipe_vision(target_move)
+			var/old_loc = user.loc
 			user.loc = target_move
```

Comment:
have you tested this using forcemove?

Reply:
I chose to keep the .loc behavior because I wasn't sure if all of the extra stuff from forcemove was really necessary when we're moving through pipes. It's more of an abstract move since we don't really want to update much more (and I'd like to look at porting /tg/'s abstract move behavior at some time in the future).

Reply:
Actually just went back to using forcemove here. Should be fine, and I've tested with that.

Reply:
so that was a mistake